lib/queue.js

```diff
@@ -21,45 +21,57 @@ metasync.ConcurrentQueue = function(
   };
 };
 
+
+const process = function () {
```

Comment:
There's no space between `function` keyword and parentheses according to our codestyle. I'd prefer it to be a named function, though.